src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -127,22 +128,35 @@ public class EvtItem extends SkriptEvent {
 				.examples("on item merge of gold blocks:",
 					 	"	cancel event")
 				.since("2.2-dev35");
+		if (HAS_PLAYER_STONECUTTER_RECIPE_SELECT_EVENT) {
+			Skript.registerEvent("Stonecutter Recipe Select", EvtItem.class, PlayerStonecutterRecipeSelectEvent.class, "stonecutting [[of] %-itemtypes%]")
+				.description("Called when a player selects a recipe in a stonecutter.")
+				.examples(
+					"on stonecutting stone slabs",
+						"\tcancel the event",
+					"",
+					"on stonecutting:",
+						"\tbroadcast \"%player% is using stonecutter to craft %event-item%!\""
+				)
+				.since("INSERT VERSION")
+				.requiredPlugins("Paper 1.16+");
```

Comment:
```suggestion
					.description("Called when a player selects a recipe in a stonecutter.")
					.examples(
							"on stonecutting stone slabs",
								"\tcancel the event",
							"",
							"on stonecutting:",
								"\tbroadcast \"%player% is using stonecutter to craft %event-item%!\""
					)
					.since("INSERT VERSION")
					.requiredPlugins("Paper 1.16+");
```